### ISSUE TITLE
How to translate <Use>?

### ISSUE THREAD

**Issue by msageryd** [react-native-community/react-native-svg#646]
I need to rotate my symbols, but I can't make out where the center of rotation is at.

Here is what I want to do:
```
<Use
  href={'#mySymbol'}
  transform={{ 
    rotation: 45,
    translateX: ?,
    translateY: ?
  }}
  x={x}
  y={y}
  width={w}
  height={h}
/>
```

I can't figure out how to translate in order to put the center of rotation in the center of the Symbol.  The Circles and the Symbols are rendered with the same center point as seen in the first two images. When I rotate the Symbol without translation I get very odd results.

Is this a bug? Or do I need to calculate the translation differently because of the scaling of the Symbol? In what order are the transforms performed? 
- scale Symbol via w*h
- rotate via transform
- translate via transform

**Rotation: 0, No translation**
![image](https://user-images.githubusercontent.com/4483085/38559502-793fbda8-3cd3-11e8-8430-32b273f47a6a.png)
![image](https://user-images.githubusercontent.com/4483085/38559512-86eaf8a0-3cd3-11e8-94e8-4a1a546b02d8.png)

**Rotation: 45, No translation**
![image](https://user-images.githubusercontent.com/4483085/38559555-9f738f7c-3cd3-11e8-848d-5a8ec9a32950.png)
![image](https://user-images.githubusercontent.com/4483085/38559734-17ea5dc8-3cd4-11e8-84a2-0ee6838e1844.png)

**Comment by msand** [comment]
There seems to be a bug here, made a simple test to demonstrate, currently it seems to have the origin in the top left corner of the referenced symbol, when it should be using the CTM.
```jsx
import React, { Component } from 'react';
import { View, StyleSheet, Dimensions } from 'react-native';
import { Constants, Svg } from 'expo';
const { Symbol, Circle, Use, Rect } = Svg;

const { width, height } = Dimensions.get('window');

const hair = 100 * StyleSheet.hairlineWidth / width;

export default class App extends Component {
  render() {
    return (
      <View style={styles.container}>
        <Svg height={height} width={width} viewBox="0 0 100 100">
          <Symbol id="symbol" viewBox="0 0 100 100" width="10" height="10">
            <Rect x="0" y="0" width="100" height="100" fill="blue" />
            <Circle
              cx="50"
              cy="50"
              r="40"
              strokeWidth="8"
              stroke="red"
              fill="red"
            />
          </Symbol>

          <Rect
            x="0"
            y="0"
            width="100"
            height="100"
            stroke="blue"
            fill="none"
            strokeWidth={hair}
          />
          <Use
            href="#symbol"
            x="0"
            y="0"
            width="20"
            height="20"
            transform="rotate(45, 10, 10)"
          />
          <Use
            href="#symbol"
            x="40"
            y="40"
            width="20"
            height="20"
            transform="rotate(-45, 10, 10)"
          />
          <Use
            href="#symbol"
            x="0"
            y="80"
            width="20"
            height="20"
            transform="rotate(-45, 10, 90)"
          />
        </Svg>
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    alignItems: 'center',
    justifyContent: 'center',
    paddingTop: Constants.statusBarHeight,
    backgroundColor: '#ecf0f1',
  },
});
```
https://snack.expo.io/H1NaP-x3z

```jsx
import React from 'react';
import { render } from 'react-dom';

const styles = {
  fontFamily: 'sans-serif',
  textAlign: 'center',
};

const App = () => (
  <div style={styles}>
    <svg height="200" width="200" viewBox="0 0 100 100">
      <symbol id="symbol" viewBox="0 0 100 100" width="10" height="10">
        <rect x="0" y="0" width="100" height="100" fill="blue" />
        <circle
          cx="50"
          cy="50"
          r="40"
          strokeWidth="8"
          stroke="red"
          fill="red"
        />
      </symbol>

      <rect x="0" y="0" width="100" height="100" stroke="blue" fill="none" strokeWidth="1" />
      <use href="#symbol" x="0" y="0" width="20" height="20" transform="rotate(45, 10, 10)" />
      <use href="#symbol" x="40" y="40" width="20" height="20" transform="rotate(45)" />
      <use href="#symbol" x="0" y="80" width="20" height="20" transform="rotate(-45, 0, 0)" />
    </svg>
  </div>
);

render(<App />, document.getElementById('root'));

```
https://codesandbox.io/s/8w85llk78

**Comment by msageryd** [comment]
Fixed in 9.0.0
